package.json

```diff
@@ -41,7 +41,7 @@
     "extensionKind": [
         "workspace"
     ],
-    "aiKey": "AIF-d9b70cd4-b9f9-4d70-929b-a071c400b217",
+    "aiKey": "0c6ae279ed8443289764825290e4f9e2-1a736e7c-1324-4338-be46-fc2a58ae4d14-7255",
```

Comment:
💡 Please note: I might have commented on this before - but what is this GUID and from where is this getting generated? worth to archive this. Thankyou!

Reply:
This is the new application insights key.